### DISCUSSION TITLE
[Feature] Manually adding persons

### DISCUSSION THREAD

**Discussion by spark108** [immich-app/immich#14629] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [x] Yes

### The feature

Add the ability to manually specify faces in a photo in addition to automatic detection. Automatic detection does not always cope with the task if there are low-quality photos, a non-standard angle of rotation of the face, or simply if the face is only in one photo.

I would like to see a "Specify face" button, which, when clicked, asks you to select the coordinate in the photo where the person is located and specify an existing person or create a new one.

### Platform

- [X] Mobile
- [X] Server
- [X] Web

**Comment by bo0tzz** [comment]
#7301
